### ISSUE TITLE
UEFN / Unreal Editor For Fortnite (App name: "Fortnite_Studio") has an error when launched with Legendary but works fine with Epic Games Launcher.

### ISSUE THREAD

**Issue by dimostheenis** [derrod/legendary#645]
UEFN / Unreal Editor For Fortnite (App name: "Fortnite_Studio") has an error when launched with Legendary but works fine with Epic Games Launcher.

## Platform
Operating system and version: Windows 11 Home 23H2
Legendary version (`legendary -V`): v.0.20.34

## Expected Behavior
UEFN launches successfully

## Current Behavior
An error that says "Invalid backend requested! Exiting." appears. (1st screenshot) Then, the initializing screen/launcher appears, but then it closes and nothing happens after, and no UEFN is anywhere to be found in the task manager.

F:\Epic>legendary launch Fortnite_Studio
[cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[Core] INFO: Getting authentication token...
[cli] INFO: Launching Fortnite_Studio...
(Then the problem appears)

## Steps to Reproduce
1. Own the Fortnite DLC named "Unreal Editor For Fortnite"
2. Install and launch it

![UnrealEditorFortnite-Win64-Shipping_PW0Rad5eko](https://github.com/derrod/legendary/assets/76966291/97ada60d-a66a-48d8-bf26-ab31ec11225a)
![UnrealEditorFortnite-Win64-Shipping_kh1ZyZZQs7](https://github.com/derrod/legendary/assets/76966291/7d4e8c2c-2caf-46b6-88a7-8af51924a852)

## Additional information
It works perfectly fine on Epic Games launcher but not on Legendary, so it's not a problem with my computer. (I believe so)

**Comment by meyverick** [comment]
Have you found a solution? I have the same error.

**Comment by cuhsy** [comment]
Nope, I had downloaded Epic Games launcher to use it, but I have deleted it and haven't used UEFN in a while, so you would have to ask someone else.
